### ISSUE TITLE
Leaving a channel as owner renders it useless

### ISSUE THREAD

**Issue by dorsax** [SamSanai/VoiceMaster-Discord-Bot#29]
What to do to recreate the issue:
1. Jointhe creation channel to create a VC
2. Join together w/ users and edit the channel via commands, e.g. `.voice limit 4`
3. switch the channel to another one 
4. switch back. You are now unable to use your voice commands, because:
![image](https://user-images.githubusercontent.com/56792918/137150367-4b5599fb-8525-416e-a89f-886f95c147ac.png)
5. Go back to the creation channel: Nothing happens.

TL;DR: Bot forgets your are the owner of a channel but remembers you own one...

**Comment by SamSanai** [comment]
I'm in the middle of a rewrite. Hoping to have that fixed alongside the rewrite. Thank you for bringing it to our attention.
